src/utils/kubeclient.ts

```diff
@@ -53,3 +55,97 @@ export function createKubeClient(config: string | AppConfig, context?: string) {
 
   return kc;
 }
+
+function parseCanI(stdout: string, namespace: string): ClusterAccess {
+  const lines = stdout.split('\n');
+
+  const permissions: KubePermissions[] = [];
+  let hasFullAccess = false;
+
+  lines.forEach((line, index) => {
+    if (!index) {
+      return;
+    }
+    const columns = line.split(/\s{2,100}/);
+
+    const [resourceName, , , rawVerbs] = columns;
```

Comment:
you could use the `pullAt` method from `lodash` here, maybe also add a comment explaining why

Reply:
I don't think that pullAt would greatly improve this code, i'd need to remove the values that I wanted and do the same thing, or remove the values that I did not want do that again